integration-tests/tests/create-application-from-public-git-source.spec.ts

```diff
@@ -5,13 +5,16 @@ import { ApplicationDetailPage } from '../support/pages/ApplicationDetailPage';
 import { ComponentPage } from '../support/pages/ComponentsPage';
 import { Applications } from '../utils/Applications';
 import { Common } from '../utils/Common';
+import { UIhelper } from '../utils/UIhelper';
 
 describe('Create Component from Public Git Source', { tags: ['@PR-check', '@publicRepo'] }, () => {
   const addComponent = new AddComponentPage();
   const componentPage = new ComponentPage();
   const applicationDetailPage = new ApplicationDetailPage();
   const applicationName = Common.generateAppName();
-  const publicRepo = 'https://github.com/dheerajodha/devfile-sample-code-with-quarkus';
+  const publicRepo = 'https://github.com/hac-test/nodejs-ex';
+  const gitReference = 'v1.0.1';
+  const contextDir = './';
```

Comment:
Can we have some real contextDir? E.g. in some other repo? I was thinking about hac-dev repo and integration-tests context dir, for example.

Reply:
Okay, you want be to create a new dir and have nodejs-ex in it and pass dir name as contextDir?

Reply:
IDK, or use some repo where we can actually test this contextDir. But if you think it is out of the scope of this PR, we can just create an issue for that :slightly_smiling_face: